contributing/documentation/overview.rst

```diff
@@ -249,21 +249,30 @@ GitHub, click on the **Show all checks** link and finally, click on the
 Build the Documentation Locally
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you have Docker installed on your machine, run these commands to build the
-docs:
+If you have Docker installed on your machine, this command will build
+the image, compile the docs, and start to watch for changes and
+automatically compile them:
 
 .. code-block:: terminal
 
-    # build the image...
-    $ docker build . -t symfony-docs
+    # We use the same user as outside to avoid permissions issues with generated files
+    $ CURRENT_UID=$(id -u):$(id -g) docker-compose up
```

Comment:
Files generated by the containers are "cache" and "build". 
Does it make sens to add such complexity to useland, to set the user's permission on files that shouldn't be edited anyway?

Reply:
If a user is not set docker will run as root and all the files and subfolders will be created for that user and whenever you switch branches or anything like that you have permission issues. That's my concern at least.

I can remove it from the basic setup and only use it when I am working on the docs as it seems to not be impacting many people and causing more issues than anything else. (also related to windows compatibility question)

Reply:
hmm I forgot about deletion when changing branch..
Does it occurs even when the folder is inside a .gitignore?

Reply:
You are right. I think I had issues on my local because I was still developing the thing. I am removing the need for the `CURRENT_UID` and closing the relevant discussions.